backend/src/openbeheer/types/_open_beheer.py

```diff
@@ -558,31 +564,36 @@ def _resultaat_as_option(resultaat: LAXResultaat, **kwargs) -> OBOption[Resultaa
     )
     label = " - ".join(label_values)
     return OBOption(
-        label=f"{label} - {resultaat.proces_type}",
+        label=f"{label}",
         value=resultaat.url,
     )
 
 
+def fetch_resultaat_options(
+    procestype_url: str | None = None,
+) -> list[OBOption[LAXResultaat]]:
+    resultaten = fetch_resultaten()
+    return [
+        as_ob_option(p)
+        for p in sorted(
+            resultaten,
+            key=lambda r: (r.proces_type, r.volledig_nummer, r.nummer, r.naam),
```

Comment:
Does it make sense to sort on proces_type (a URI) and then the other fields?

Edit: i forgot that you have many selectielijst resultaten related to a procestype.